shaders/SelectDistinctFromBackgroundColor.shader

```diff
@@ -0,0 +1,17 @@
+shader_type canvas_item;
+
+uniform vec4 bg_col : hint_color;
```

Comment:
please don't abbreviate "colour"

Reply:
speaking of colo(u)r: are there any guidelines on british vs american english?

Reply:
Use British English, unless something like Godot requires to use American spelling.